publication-commons/src/main/java/no/unit/nva/publication/model/business/FileEntry.java

```diff
@@ -42,6 +53,25 @@ public static FileEntry create(File file, SortableIdentifier resourceIdentifier,
                              userInstance.getTopLevelOrgCristinId(), userInstance.getCustomerId(), file);
     }
 
+    public static FileEntry queryObject(UUID fileIdentifier, SortableIdentifier resourceIdentifier) {
```

Comment:
This appears to only be used in tests, is that correct? Could move it to a test util in that case?

Also, could you rename it to something more  descriptive?

Reply:
Yes, all this methods are not used in production yet, it will be the next step (or one of next steps).